packages/babel-plugin-syntax-decorators/test/index.js

```diff
@@ -0,0 +1,49 @@
+import { parse } from "@babel/core";
+import syntaxDecorators from "../lib";
+
+describe("legacy option", function() {
+  const oldSyntax = "@dec export class Foo {}";
```

Comment:
https://github.com/atlassian/jest-in-case could make those assertions nicely grouped